distribution/Dockerfile

```diff
@@ -0,0 +1,28 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM centos:7
+ARG VERSION
+
+RUN yum install -y java-1.8.0-openjdk-devel which ; yum clean all ; rm -rf /var/cache/yum
+
+COPY target/apache-drill-$VERSION.tar.gz /tmp
+RUN mkdir /opt/drill
+RUN tar -xvzf /tmp/apache-drill-$VERSION.tar.gz --directory=/opt/drill --strip-components 1
+
+ENTRYPOINT /opt/drill/bin/drill-embedded
```

Comment:
Will user be able to use sqlline if drill if already started in embedded mode? Also how to stop drill in this case?

Reply:
When user uses run command (without -d / --detach), he will get Sqlline. But if he restarts the container (stop + start) or runs container with -d / --detach, he won't get Sqlline. In this case he needs to use `drill-localhost` command to access Sqlline. I think we need to document this.

Reply:
But drill-localhost will not work right? That requires support for standalone mode - which requires ZK. Or am I missing something? 

Yes, with the current approach, the container is only alive until the embedded sqlline session is active. If they exit and reattach, they will get a new session (all history is lost). 

I could add an init script with an infinite wait, and suggest going the docker exec route, to keep the container alive and maintain history - but since it would still only support embedded mode (until we add support for standalone using ZK), i went with this approach. What do you suggest?

One possibility, is we can document instructions on how to make changes to get what I suggested above working.

Reply:
Actually, it worked. I have started drill-embedded in one console. And in other connected to drill using drill-localhost.

Reply:
It worked for me too! I think I understood why - drill-localhost connects to a locally running Drillbit (doesn't matter embedded or distributed). Since the run command which stars Drill in embedded mode keeps the Drillbit running, we can use drill-localhost to launch another session. Cool, i'll document this.